cmd/analyze_deps.go

```diff
@@ -26,6 +26,10 @@ to quickly create a Cobra application.`,
 		if err != nil {
 			return err
 		}
+		cyclesMode, err := cmd.Flags().GetBool("cycles")
```

Comment:
what do you think about binding the flags to vars like 
```
rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.cobra.yaml)")
```

see example in readme here https://github.com/spf13/cobra

Reply:
Thanks for this! I will look into it. I'm currently working on the UX and we're probably adding cycles as another sub-command but I will take care of this when adding flags to other commands :)